src/bulk_insert/bulk_insert.c

```diff
@@ -7,388 +7,211 @@
 
 #include "bulk_insert.h"
 #include "../stores/store.h"
+#include <errno.h>
 #include <assert.h>
-
-typedef struct {
-    long long node_count;   // Number of nodes to create under this label.
-    const char *label;      // Label.
-    int label_id;           // ID given to label, recognized by graph object.
-    int attribute_count;    // How may attributes are there for this label.
-    char **attributes;      // Array of attributes.
-} LabelDesc;
-
-void _BulkInsert_Reply_With_Syntax_Error(RedisModuleCtx *ctx, const char* err) {
-    RedisModule_ReplyWithError(ctx, err);
+#include <libgen.h>
```

Comment:
Why do we include libgen?

Reply:
That was a leftover from the file-passing approach, deleting it.